### DISCUSSION TITLE
Can translations be shared across projects?

### DISCUSSION THREAD

**Discussion by JulianGro** [WeblateOrg/weblate#7121] [Q&A] [answered]
Hello,

we have two projects that forked. Can we share translations somehow?
As in, if a string was translated on one Weblate project, can the translation automatically be used or suggested on the other Weblate project that has the same string?

**Comment by nijel** [comment, marked as answer]
You can use automatic translation for that. Either manually or via add-on.

**Reply by JulianGro** [reply]
Thanks. That is exactly what I was looking for.
